RedHat_NLP/Deliverable 1.ipynb

```diff
@@ -0,0 +1,285 @@
+              "ok": true,
+              "headers": [
+                [
+                  "content-type",
+                  "application/javascript"
+                ]
+              ],
+              "status": 200,
+              "status_text": ""
+            }
+          },
+          "base_uri": "https://localhost:8080/",
+          "height": 407
+        },
+        "outputId": "a48fb16f-e372-4715-fa68-25c9241a3437"
+      },
+      "source": [
+        "# Web scraping (Kyle, due 02/26/2021)\n",
+        "#     BeautifulSoup for web scraping\n",
+        "\n",
+        "from urllib.request import urlopen\n",
+        "from bs4 import BeautifulSoup\n",
+        "from google.colab import files\n",
+        "import pandas as pd\n",
+        "import io\n",
+        "import numpy as np\n",
+        "import dateutil\n",
+        "from dateutil import parser\n",
+        "import textblob\n",
+        "from textblob import TextBlob\n",
+        "import nltk\n",
+        "from textblob import Word\n",
+        "import matplotlib.pyplot as plt\n",
+        "from nltk.stem.snowball import SnowballStemmer\n",
+        "nltk.download('punkt')\n",
```

Comment:
You might want to break the work up into a couple more cells for increased readability. See [this notebook](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/notebooks/data-sources/TestGrid/testgrid_EDA.ipynb) for an example. 

There is no strict rules for how to break up code in notebooks, but a good rule of thumb is like, every logical step where you complete a set of data transformations, or generate some new output, put it in its own cell and add a markdown cell below it annotating what the cell above does and why. Again, see the linked notebook for an example of what I'm talking about : )